irc/user_test.ts

```diff
@@ -0,0 +1,35 @@
+import { Conn } from "deno";
+import { assert, test } from "./package.ts";
+import { User } from "./user.ts";
+
+// 
+test(function userModes() {
+  // use empty object, as we're not testing connections here
+  const user = new User({} as Conn);
+  assert(!user.isInvisible);
+  assert(!user.isOp);
+  assert(!user.isLocalOp);
+  assert(!user.isWallops);
+
+  user.isInvisible = true;
+  user.isOp = true;
+
+  assert(user.isInvisible);
+  assert(user.isOp);
```

Comment:
Sure. These tests are somewhat vacuous, but I guess it exercises that the properties are available.